staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -221,16 +221,6 @@ var (
 		},
 		[]string{priorityLevel, flowSchema},
 	)
-	apiserverRequestConcurrencyLimit = compbasemetrics.NewGaugeVec(
```

Comment:
Let's not delete the metric in this large PR - please move it to a separate PR (with explicit approval from sig instrumentation folks).

Reply:
The problem here is that this metric means two things (NominalCL, CurrentCL) that are equal before this PR but not generally equal after this PR.  Keeping it would be, at best, misleading.  And which meaning would be best to keep for this metric?

Reply:
@logicalhan , @dgrisonnet : any thoughts here?

Reply:
I agree with @wojtek-t, removing the metric in a separate would make it easier to track and document in a release note.

Reply:
>  And which meaning would be best to keep for this metric?

Let's open the PR removing it now and merge that right after this PR merges.
If we do that - we can choose whatever meaning you prefer.

Reply:
I'm okay with either decision.

Reply:
No!  The problem occurs right here in this PR.  This PR make a priority level's configured concurrency limit not necessarily  the same as the concurrency limit that is enforced.

Reply:
Can you amend the release note to include the removal of this metric?